src/server/qgsserverrequest.h

```diff
@@ -260,6 +260,12 @@ class SERVER_EXPORT QgsServerRequest
     void setBaseUrl( const QUrl &url );
 
   private:
+
+    /**
+     * Initialize the mRequestHeaderConv content
```

Comment:
Can you make this a bit more verbose (variable names are an implementation detail, doc strings should be on a functional level)